modules/nf-core/nanofilt/main.nf

```diff
@@ -0,0 +1,51 @@
+process NANOFILT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nanofilt:2.8.0--py_0':
+        'biocontainers/nanofilt:2.8.0--py_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val   readlength
+    val   readqual
```

Comment:
If I understand the tool correctly, these are not mandatory arguments, but optional and so should be supplied through the `ext.args` variable. Only `val` types that must be supplied should be under `input:`.

Reply:
An extra `path` is needed here for the optional summary file.

Reply:
I made those changes in the latest commit